templates/repo/issue/list.tmpl

```diff
@@ -34,6 +34,15 @@
 		<div id="issue-actions" class="issue-list-toolbar gt-hidden">
 			<div class="issue-list-toolbar-left">
 				{{template "repo/issue/openclose" .}}
+				<!-- Total Tracked Time -->
```

Comment:
Could you put this in tempate.

Reply:
I dont think so ... else we would also have to do it with label filters, etc ...

As it would only dedup 2 sml codeblocks to one.
the better option would be to dedup the whole menue

Reply:
(Witch is outside of this scope)

Reply:
> I dont think so ... else we would also have to do it with label filters, etc ...
> 
> 
> 
> As it would only dedup 2 smal codeblocks to one.
> 
> the better option would be to dedup the whole menue

No worries

Reply:
@puni9869 still thanks for pointing out, in general we should use subtemplates way more frequent ;)